server/routes/index.ts

```diff
@@ -163,7 +163,33 @@ export default function routes(services: Services): Router {
       next(err)
     }
   })
+  use('/addToYourCases/', async (req: Request, res: Response, next) => {
```

Comment:
Sorry to be a pain but could this be moved into its own Controller? We are planning to move everything from this file into their own controllers at some point in the future. Also it might be worth having this as a POST rather than a GET since it alters data in the database.

If you need an example have a look at `addAppointmentController.ts`